Automattic/wp-calypso
# Help Center: Hardcode Contact Channels CTA colors

<img width="412" alt="Screenshot 2023-03-09 at 19 50 02" src="https://user-images.githubusercontent.com/17054134/224125657-de31a724-1849-41eb-be18-ce8fdcc767d5.png">

Related to p1678102495293989-slack-C02T4NVL4JJ

## Proposed Changes

This hardcodes the CTA colors of the Help Center across all envs.

## Testing Instructions

- Login to https://hud-staging.happychat.io/ and make yourself available.
- Open the Help Center, all options should have different nice colors.
- Try in Calypso, and in /wp-admin, colors should be the same.
